guide/C/ch_invest.xml

```diff
@@ -975,22 +972,22 @@ Income
     <title>Buying Shares</title>
 
     <sect2 id="invest-buy-stock2">
-      <title>Entering Preexisting Shares</title>
+      <title id="invest-buy-stock2.title">Entering Preexisting Shares</title>
 
-      <para>The examples in this section use <guilabel>Transaction Journal </guilabel> view.
+      <para>The examples in this section use <guilabel>Transaction Journal</guilabel> view.
       </para>
 
-      <para>To register the initial 100 shares of this stock that you purchased previously, on the first
-        (transaction) line, enter the date of the purchase (eg: Jan. 1 2005) and
-        <guilabel>Description</guilabel> (eg: Opening Balance). On the first split line, enter 100
-        in <guilabel>Shares </guilabel>, delete the (unit) <guilabel>Price</guilabel> (it will be
-        calculated when you <keycap>Tab</keycap> out of the split) and enter 2000 in the
+      <para>To register the initial <userinput>100</userinput> shares of this stock that you purchased previously, on the first
+        (transaction) line, enter the date of the purchase (for example, Jan. 1 2005) and
+        <guilabel>Description</guilabel> (for example, Opening Balance). On the first split line, enter <userinput>100</userinput>
```

Comment:
> IMHO most brackets should be replaces as bad style. Keep in mind that most authors were no native English speakers.
I think so, too.
It is very difficult for me to translate them and long sentenses into Japanese.

If a stenense is written like "A: is B, and C, if D", we will transrate like "D, C, A, B". So some block tags prevent me from translation. That's why I prefer variablelist.

Reply:
Plenty of native English speakers overuse parentheses and get commas wrong. Style guides are full of warnings about that and most agree with @fellen that parentheses should be used as little as possible. Long sentences are another common failing of native English speaking writers and they're hard for everyone to read. Short declarative sentences are the best style.

@usamimikamen-sk Aside from the Italian translation the documentation is not at present translated with gettext so were you to begin updating the Japanese translation you'd be working more as a book translator where you'd read and understand a whole section of English and rewrite it in idiomatic Japanese rather than making a chunkwise translation as you'd do when translating a program's strings.

As for dates I agree that YMD is the best short form but I think it's better in narrative to spell out the month name. Ideally all of the screenshots would be redone with YMD and all of the descriptions with day month-name year. Americans are pretty much the only people still using month-name day, year and they can easily parse day-first dates.

Reply:
In preface,
about FDL, not investment advice, local tax lows, date format, and so on.

Reply:
As a result, the date format is 1 Feb. 2021 in the English document.
In the image, it is 2020/02/01. (yyyy/mm/dd)
Is this OK?

Reply:
Right!

I wonder if ye should add a section about our conventions to the preface of the book.